backend/prompt_studio/prompt_studio_index_manager_v2/models.py

```diff
@@ -0,0 +1,130 @@
+import json
+import logging
+import uuid
+
+from account_v2.models import User
+from django.db import models
+from django.db.models.signals import pre_delete
+from django.dispatch import receiver
+from prompt_studio.prompt_profile_manager_v2.models import ProfileManager
+from prompt_studio.prompt_studio_core_v2.prompt_ide_base_tool import PromptIdeBaseTool
+from prompt_studio.prompt_studio_document_manager_v2.models import DocumentManager
+from unstract.sdk.constants import LogLevel
+from unstract.sdk.vector_db import VectorDB
+from utils.models.base_model import BaseModel
+from utils.user_context import UserContext
+
+logger = logging.getLogger(__name__)
+
+
+class IndexManager(BaseModel):
```

Comment:
Don't we need org info here? @muhammad-ali-e

Reply:
@gaya3-zipstack I don't see any direct relations with Organizations.  IF we need any we can add once iut tested

Reply:
ref discussion: https://github.com/Zipstack/unstract/pull/471

Reply:
I am assuming since profile_manager_id is there, we can separate out the records in this table Might have to take a look at it later.